spec/models/grantee_spec.rb

```diff
@@ -28,4 +28,11 @@
       expect(subject.region).to eq hses_grantee[:attributes][:region]
     end
   end
+
+  # TODO: test for missing attribute
```

Comment:
[suggestion] You could potentially put a pending test here as a reminder, although I guess I don't know how the GitHub Actions task would respond to that....

https://relishapp.com/rspec/rspec-core/v/2-0/docs/pending/pending-examples